src/evo/simplifiedmns.h

```diff
@@ -74,6 +74,9 @@ class CSimplifiedMNListEntry
                 obj.isValid
                 );
         if (obj.nVersion == BASIC_BLS_VERSION) {
+            if ((s.GetType() & SER_NETWORK) && s.GetVersion() < DMN_TYPE_PROTO_VERSION) {
+                return;
+            }
```

Comment:
Isn't the idea here to be able to do legacy scheme to old peers?

Reply:
I mean, `obj.nVersion == BASIC_BLS_VERSION` part is on the object level so to say and `s.GetVersion() < DMN_TYPE_PROTO_VERSION` is on the p2p protocol level. would probably make sense to avoid mixing the two when it's not really needed.

Reply:
@PastaPastaPasta No I don't think we should. `MerkleRootMNList` won't match.
The idea here is to simply allow old mobile clients to correctly parse `mnlistdiff` messages without banning nodes.